### PR TITLE
MGMT-1203: Set StatusUpdatedAt upon cluster/host create

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -274,6 +274,7 @@ func (b *bareMetalInventory) RegisterCluster(ctx context.Context, params install
 		ServiceNetworkCidr:       swag.StringValue(params.NewClusterParams.ServiceNetworkCidr),
 		SSHPublicKey:             params.NewClusterParams.SSHPublicKey,
 		UpdatedAt:                strfmt.DateTime{},
+		StatusUpdatedAt:          strfmt.DateTime(time.Now()),
```
Comment (filanov) [inline internal/bminventory/inventory.go:277]: shouldn't it be part of the internal state machine logic?

Comment (avishayt) [reply internal/bminventory/inventory.go:277]: It already is - the problem is that it's not set upon creation (before state machine is in action)

Comment (filanov) [reply internal/bminventory/inventory.go:277]: if it's only on registration then you can add it here: https://github.com/filanov/bm-inventory/blob/master/internal/host/transition.go#L53 just before the create call

**Comment by avishayt** [comment]
@filanov Makes sense, is this better?

**Pull request merged**